src/freertos_drivers/common/RailcomDriver.hxx

```diff
@@ -74,6 +77,7 @@ class NoRailcomDriver : public RailcomDriver {
   void start_cutout() OVERRIDE {}
   void middle_cutout() OVERRIDE {}
   void end_cutout() OVERRIDE {}
+  void no_cutout() OVERRIDE {}
```

Comment:
Please add Doxygen Header. Could use a "cond IGNORE" block tag if you prefer no suppress Doxygen warnings to keep these definitions inline and brief since the comment could be considered trivial, already covered in the base class.

Reply:
I found this: https://www.doxygen.nl/manual/config.html#cfg_inherit_docs

Based on that if I do add any comment here, I will defeat the purpose of inheriting docs. I generated the docs and this is what I got for NoRailcomDriver:

![image](https://user-images.githubusercontent.com/3719690/96347066-77c25100-109f-11eb-843b-c854ae44331c.png)

so it seems to be working as expected.